### PR TITLE
Introduce ykfde-format

### PR DESCRIPTION
**Pull request by Vincent43**
This will add missing piece in our scripts: ability to create new containers with yubikey challenge-response passphrase used by default.

We already cover adding passphrases to existing containers and opening created containers so this will be nice addition.

### CODE REVIEW

File: src/ykfde-format [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+_tmp="$(ykinfo -"$YKFDE_CHALLENGE_SLOT" 2>&1)";
+[ "$DBG" ] && printf '%s\n' "$_tmp"
+
+echo "   Remember to touch the device if necessary."
+[ "$DBG" ] && printf '%s\n' "   Running: ykchalresp -$YKFDE_CHALLENGE_SLOT \"$YKFDE_CHALLENGE\"..."
+_passphrase="$(ykchalresp -"$YKFDE_CHALLENGE_SLOT" "$YKFDE_CHALLENGE" 2>/dev/null| tr -d '\n')"
+
+if [ -z "$_passphrase" ]; then
+    echo "   Yubikey did not provide a response - Initializing second attempt, touch the device if necessary."
+    _passphrase="$(ykchalresp -"$YKFDE_CHALLENGE_SLOT" "$YKFDE_CHALLENGE" 2>/dev/null| tr -d '\n')"
```
Comment (agherzan) [inline src/ykfde-format]: What happens if you get no response three times? Wouldn't that use an empty password for cryptsetup?

Comment (Vincent43) [reply src/ykfde-format]: Apparently yes. :smile:  I'll fix that later.

**Comment by Vincent43** [comment]
I added fail-safe when Luks passphase is empty. `ykfde-enroll` script was also affected.

**Comment by agherzan** [comment]
@Vincent43 cheers!

**Pull request merged**